features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml

```diff
@@ -88,10 +88,10 @@
                 <tr>
                     <td colspan="2" align="center" style="padding: 20px 40px 40px 40px;" bgcolor="#f0f0f0">
                         <p style="font-size: 12px; margin: 0; line-height: 24px; font-family: 'Nunito Sans', Arial, Verdana, Helvetica, sans-serif; color: #777;">
-                            &copy;2020
+                            &copy;2022
```

Comment:
If possible, it would be better to template the year

Reply:
Fixed